src/Sylius/Component/Attribute/Model/AttributeSubjectInterface.php

```diff
@@ -28,10 +28,15 @@ public function getAttributes(): Collection;
     /**
      * @param string $localeCode
      * @param string $fallbackLocaleCode
+     * @param string|null $baseLocaleCode
      *
      * @return Collection|AttributeValueInterface[]
      */
-    public function getAttributesByLocale(string $localeCode, string $fallbackLocaleCode): Collection;
+    public function getAttributesByLocale(
+        string $localeCode,
+        string $fallbackLocaleCode,
+        ?string $baseLocaleCode = null
```

Comment:
Maybe we could start adding TODO messages with the information, that from 2.0 the third parameter would be mandatory?

Reply:
If we intend to make it required then `E_USER_DEPRECATED` error should be triggered if value was not passed. To be honest, I'd rather make fallback locale code optional as well (there's no method to get product attributes values in given locale only now).